src/components/VirtualizedList/VirtualizedList.tsx

```diff
@@ -386,7 +394,7 @@ const VirtualizedList: VibeComponent<VirtualizedListProps> = forwardRef(
             updateListSize(width, height);
             return (
               <List
-                ref={mergedListRef}
+                ref={mergedListRef as unknown as LegacyRef<VariableSizeList<unknown>>}
```

Comment:
```suggestion
                ref={listRef}
```
Cause it's anyway inside `mergedListRef`